relay-server/src/actors/store.rs

```diff
@@ -1023,11 +1024,21 @@ struct ProfileKafkaMessage {
     payload: Bytes,
 }
 
+#[allow(dead_code)]
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "snake_case")]
+enum CheckInMessageType {
```

Comment:
The first 2 sentences of your issue would be nice here. Documenting the use of the values and that relay is only supposed to ever send `CheckIn`'s

Reply:
I'll add it in in a diff PR 👍